src/java.base/share/classes/java/lang/foreign/package-info.java

```diff
@@ -27,7 +27,7 @@
 /**
  * <p>Provides low-level access to memory and functions outside the Java runtime.
  *
- * <h2>Foreign memory access</h2>
+ * <h2 id="fma">Foreign memory access</h2>
```

Comment:
I would refrain from adding `id` if the section is never referred to.

Reply:
The idea is that a user might want to link to this section of the javadoc (using the new copy button). For that to be possible, the section needs an `id`. That's why I've added one to all headers.

Reply:
I see what you mean - at the same time I don't see being done anywhere else in JDK, so perhaps can wait until we settle on a more uniform solution?